src/test/java/sonarquberepair/processor/spoonbased/CastArithmeticOperandTest.java

```diff
@@ -14,10 +14,16 @@
     public void test() throws Exception {
         String fileName = "CastArithmeticOperand.java";
         String pathToBuggyFile = Constants.PATH_TO_FILE + fileName;
-        String pathToRepairedFile = "./spooned/" + fileName;
+        String pathToRepairedFile = Constants.WORKSPACE + "/spooned/" + fileName;
 
+        System.out.println(pathToBuggyFile + " " + pathToRepairedFile);
```

Comment:
Why does this line exist?

Reply:
Debug message from before. I have removed it :) .